### DISCUSSION TITLE
bzlmod loads more packages than in workspace mode

### DISCUSSION THREAD

**Discussion by luzhanning** [bazelbuild/bazel#21997] [Q&A] [answered]
Recently I just switched my GO project to bzlmod, but I discovered for the same target, bzlmod loads more packages than origin. For same target, builds without cache now takes much longer

**Comment by meteorcloudy** [comment]
Can you provide a project to reproduce this?

/cc @fmeum Anything you have in mind that would cause this?

**Reply by fmeum** [reply]
I don't. @luzhanning Could you share a Starlark profile recorded with `--profile` or, ideally, a self-contained reproducer?

**Comment by luzhanning** [comment, marked as answer]
Never mind, just find out the root cause, not related to bzlmod.  I will close this issue
